.github/workflows/docs.yml

```
@@ -0,0 +1,146 @@
+          python -m pip install -r docs/requirements.txt
+          python -m pip install aiofiles cryptography
+
+      - name: Build documentation
+        run: |
+          cd docs
+          make html
+        env:
+          PYTHONPATH: ${{ github.workspace }}
+
+      - name: Deploy to GitHub Pages
+        uses: peaceiris/actions-gh-pages@v3
```

actionlint (1.7.7)

the runner of "peaceiris/actions-gh-pages@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)